### ISSUE TITLE
Redirect vocabustudyonline.web.app to vocabustudy.org

### ISSUE THREAD

**Issue by IMGROOT2** [For-0/vocabustudy#230]
in case people use the old URL

**Comment by grimsteel** [comment]
Won't fix - vocabustudy.org is blocked 😠

I might do this in the future if I can get it unblocked (I have contacts)

**Comment by grimsteel** [comment]
I can't do a real redirect (firebase default domain restrictions) but I can do some stuff with JS
